src/gemm_v2.cpp

```diff
@@ -324,6 +326,173 @@ inline void ProfilingRecordStop(const Handle& handle, HipEventPtr& start, HipEve
     handle.ResetKernelTime();
     handle.AccumKernelTime(mS);
 }
+
+#define HIP_BLAS_STATUS_CHECK(cmd)                       \
+    do                                                   \
+    {                                                    \
+        hipblasStatus_t e = cmd;                         \
+        if(e != hipblasStatus_t::HIPBLAS_STATUS_SUCCESS) \
+        {                                                \
+            return e;                                    \
+        }                                                \
+    } while(0)
```

Comment:
Thoughts on the updated version @CAHEK7?

I made the changes, but still required a macro to make it easy to call the templated method that throws when status isn't success. I think this was the cleanest way to do this, but curious if you had a suggestion that would be better.

Thanks!

Reply:
It can be something like this:
```C++
static void hipblas_status_check(hipblasStatus_t status)
{
    if(status != hipblasStatus_t::HIPBLAS_STATUS_SUCCESS)
    {
        MIOPEN_THROW(miopenStatusInternalError, "hipBlasLt error encountered");
    }
}

hipblas_status_check(gemm.setProblem(...));
```
THere is no need to pass whole function into the checker, we need just a status.

Reply:
hehe good point! 

I will update. Thanks!